tests/token.test.ts

```diff
@@ -209,5 +238,44 @@ describe.each([{ permission: 'Private' }])(
         searchClient.index(UID).search('pride')
       ).rejects.toHaveProperty('code', 'invalid_api_key')
     })
+
+    test(`${permission} key: Search in tenant token with expired token`, async () => {
```

Comment:
to me is more like: `creates tenant token with an expiration date in the past throws an error`